### ISSUE TITLE
Calling token.getWithoutPrompt is failing without sessionToken provided

### ISSUE THREAD

**Issue by kdstew** [okta/okta-auth-js#177]
When calling
```
token.getWithoutPrompt({
  responseType: 'token',
  scopes: ['email']
}, {
  authorizeUrl: '...',
  userinfoUrl: '...',
  issuer: '...'
})
```
Within `token.getWithoutPrompt` [here](https://github.com/okta/okta-auth-js/blob/master/lib/token.js#L460) `display` is set to `null`.  Then the merged authOptions are passed to `token.getToken`.  Then [here](https://github.com/okta/okta-auth-js/blob/master/lib/token.js#L328) `getDefaultOAuthParams` is called with authOptions and [here](https://github.com/okta/okta-auth-js/blob/master/lib/token.js#L213) `removeNils` is called on the authOptions.  This results in the `display: null` being removed from authOptions.  So when [this line](https://github.com/okta/okta-auth-js/blob/master/lib/token.js#L360) is reached the check fails because `display` has been removed from the authOptions.  The `flowType` gets set to `IMPLICIT` and I end up with [this error](https://github.com/okta/okta-auth-js/blob/master/lib/token.js#L451) being thrown.

Note that in this scenario `oauthOptions.sessionToken` is not set, but there is an active session cookie available.

**Comment by kdstew** [comment]
It looks like the `util.removeNils` was added in the most recent version in this PR https://github.com/okta/okta-auth-js/pull/171/files#diff-398ed3b014436a5204583323ea29320bR213

**Comment by kdstew** [comment]
Thanks @jmelberg-okta!

**Comment by jmelberg-okta** [comment]
Sorry for the bug - this issue has been resolved with [`@okta/okta-auth-js@2.2.0`](https://github.com/okta/okta-auth-js/releases/tag/%40okta%2Fokta-auth-js%402.2.0).

**Comment by kdstew** [comment]
Thank you for the quick fix @jmelberg-okta!
